hardhat-tests/package.json

```diff
@@ -53,10 +53,12 @@
     "eslint-plugin-prettier": "3.4.0",
     "ethereumjs-abi": "^0.6.8",
     "ethers": "^6.1.0",
+    "fs-extra": "11.2.0",
     "hardhat": "2.22.5",
     "mocha": "^10.0.0",
     "prettier": "2.4.1",
     "rimraf": "^3.0.2",
+    "semver": "7.6.0",
```

Comment:
What are these dependencies used for?

Reply:
These were peer dependencies that ESLint warned me against. This is already in main as per https://github.com/NomicFoundation/edr/commit/048c0c0524ca785c0ea9b9f163887b3473903376